app/views/banners/index.html.erb

```diff
@@ -43,6 +44,13 @@
                     No
                   <% end %>
                 </td>
+                <td class="min-width">
+                  <% if banner.expires_at %>
+                    <%= distance_of_time_in_words(Time.now, banner.expires_at) %>
+                  <% else %>
+                    No
+                  <% end %>
```

Comment:
A great candidate for a method in the model or view helper (with tests).

Reply:
Moved this to `BannerHelper` which also helped me realize I was missing the case for when the expiration had passed, so I fixed that too